scripts/deploy_docker.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+PROJECT_ID=lightstep
+DATETIME=$(date "+%Y-%m-%d_%H-%M-%S")
+BASENAME=lightstep-benchmark
+PROJECT_ID=$(gcloud config --format text list project | cut -d' ' -f2)
+
+NAME=gcr.io/${PROJECT_ID}/${BASENAME}
+
+docker build --no-cache -t ${NAME}:${DATETIME} ${GOPATH}/../docker
+docker tag -f ${NAME}:${DATETIME} ${NAME}:latest
+
+gcloud docker push ${NAME}:${DATETIME}
```

Comment:
I'm not sure what my plans are either. I just wanted to get it running for the sake of having data while the plan gels.